WordPress/Classes/ViewRelated/Blog/Blog Dashboard/BlogDashboardViewController.swift

```diff
@@ -4,12 +4,13 @@ import WordPressShared
 typealias DashboardCollectionViewCell = UICollectionViewCell & Reusable & BlogDashboardCardConfigurable
 
 protocol BlogDashboardCardConfigurable {
-    func configure(blog: Blog, viewController: BlogDashboardViewController?, apiResponse: BlogDashboardRemoteEntity?)
+    func configure(blog: Blog, viewController: BlogDashboardViewController?, apiResponse: BlogDashboardRemoteEntity?, cardType: DashboardCard)
```

Comment:
Is your plan to leave the `cardType` here?

I ask that because it seems a little bit odd. If I'm new to this code I would expect a 1:1 to map on `DashboardCard` for `enum ~> cell`. In this case, here we changed the protocol signature for this specific case (a specific cell needs to know which card it should represent).

Do you think we could split the different types of posts cards into different cells? We could use composition or inheritance. This way we would have `DashboardDraftCardCell`, `DashboardScheduledCardCell`, `DashboardFirstPostCardCell` and `DashboardNextPostCardCell`.

Also, by doing that you can get rid of `presentedPostStatus`on the cell.

While maybe this means more lines of code, I think it's clearer. Thoughts?

Reply:
That's a great point! I went with that initially cause I felt there's no need to create multiple cell classes. However, now I feel that would be much clearer.

I've changed this in 84faa3647c2f67772ba380d3ec20ee1bbe709e0f

I used inheritence and kept `DashboardPostsListCardCell` almost the same. The new subclasses are only responsible for passing the card type to `DashboardPostsListCardCell` instead of including it in the protocol.

> Also, by doing that you can get rid of presentedPostStatuson the cell.

I don't think we can do that. `presentedPostStatus` is needed by `BlogDashboardViewController` not the cell. It's needed because that VC needs to conform to `EditorAnalyticsProperties`.